### DISCUSSION TITLE
ReactiveCommand vs RelayCommand

### DISCUSSION THREAD

**Discussion by elveejay1** [AvaloniaUI/Avalonia#8893] [Q&A]
When working with WPF, I have previously used the relatively simple RelayCommand. Is there anything that should force me to switch to ReactiveCommand when using Avalonia? My code should run in both Windows and Linux (the latter being the reason to switch to Avalonia).

ReactiveCommand doesn't seem as streamlined as RelayCommand IMHO, so I'd rather stay with that.

**Comment by Mrxx99** [comment]
You can use what you prefer

**Reply by timunie** [reply]
Indeed. i personally use RelayCommand from the Community MVVM Toolkit via their source generators and it works. You can even directly bind to void, but I don't use this often as I like commands more. 

Happy coding
Tim

**Reply by elveejay1** [reply]
What do you use instead of System.Windows.Input.CommandManager, then?

**Reply by timunie** [reply]
Nothing tbh. In case I need to bind to a void in the Control, I can just do it in Avalonia.
